File: Python Exams/Basic Exam Collection/Basics Online Exam - 15 and 16 June 2019/02. Movie Day/02. Movie Day.py
picture_time, count_chapters, cad_time = [int(input()) for _ in range(3)]

preparation = picture_time * 0.15
operators_time = count_chapters * cad_time
needed_time = preparation + operators_time
total = round(abs(picture_time - needed_time))

if needed_time <= picture_time:
    print(f'You managed to finish the movie on time! You have {total} minutes left!')
else:
    print(f'Time is up! To complete the movie you need {total} minutes.')

'''
135
5
20
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 15 and 16 June 2019/02. Movie Day/02. Movie Day.js
function movieDay(pictureTime, countChapters, cadTime) {
    let preparation = pictureTime * 0.15
    let operatorsTime = countChapters * cadTime
    let neededTime = preparation + operatorsTime
    let total = Math.round(Math.abs(neededTime - pictureTime))

    if (neededTime <= pictureTime)
        return `You managed to finish the movie on time! You have ${total} minutes left!`
    return `Time is up! To complete the movie you need ${total} minutes.`
}

// console.log(movieDay(120, 10, 11))